nestedbox_core.entity.inc

```diff
@@ -53,7 +53,7 @@ function nestedbox_core_entity_info() {
       // but without the ID suffix. (In fact, you can set
       // entity_operations_entity_uri() as your URI callback, which will use the
       // value here).
-      'path' => 'admin/content/nestedbox',
+      'path' => 'entity_operations_entity_uri',
```

Comment:
You can look at EntityOperationsDefaultUIController::__construct() to see how the 'path' is used.